crates/task-impls/src/quorum_vote.rs

```diff
@@ -628,7 +628,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> QuorumVoteTaskState<TYPES, I
                     .vid_shares
```

Comment:
should this `storage.write()` on 621 also be moved right before the quorum vote task's QuorumVoteSend?

Reply:
Yes, good catch.  I didn't pay enough attention to the new task, I'll update with this change

Reply:
Can we update the changes to the `Poll` and `CancelPoll` as well?